src/client/components/MealList.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import Meal from "./Meal";
+import "./meal.css";
+const mealsUrl = process.env.REACT_APP_MEALS_URL;
+
+function MealList() {
+  const [meals, setMeals] = useState([]);
+
+  useEffect(() => {
+    try {
+      async function fetchMeals() {
+        const response = await fetch(`${mealsUrl}meals`);
+        const data = await response.json();
+
+        setMeals(data);
+      }
+      fetchMeals();
+    } catch (err) {
+      console.error(err);
+    }
+  }, []);
+
+  return (
+    <div>
+      <h2>List of meals</h2>
+      <div className="continer">
+        {meals.map((meal) => (
```

Comment:
Consider the case where meals are not fetched yet, so that this does not throw an error 💡